recipe/meta.yaml

```
@@ -25,7 +25,7 @@ requirements:
     - setuptools
   run:
     - python
-    - greenlet !=0.4.17
+    - greenlet !=0.4.17  # [py<313 and (aarch64 or x86_64)]
```

```suggestion
    - greenlet !=0.4.17  # [py<313]
```
Skipping `py<313` will be enough. See https://github.com/sqlalchemy/sqlalchemy/commit/62c242b78dee306738a2cd22f548679e9818a1ac and https://github.com/sqlalchemy/sqlalchemy/discussions/11417